src/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -57,7 +57,7 @@ public with sharing class AFFL_AccRecordType_TEST {
     public static void changePrimaryAffiliationRecordType() {        
         setup();
 
-        Contact contact = new Contact(FirstName = 'Test', LastName = 'Testerson');
+        Contact contact = UTIL_UnitTestData_TEST.getContact();
```

Comment:
Just a quick suggestion maybe, since many test methods is creating contact, would it be nice just make a static variable on top of the test class?

Reply:
That is a good suggestion. 
I am thinking that:  Does it make sense to not using a class level static variable since the purpose of using the UTIL class is to make sure every test method has unique Account/Contact?